Jenkinsfile

```diff
@@ -37,6 +37,7 @@ node('build-slave') {
                 echo "build_tag: " + build_tag
 
                 stage('Build') {
+                    sh """chmod +x build.sh"""
```

Comment:
you can merge this line to the below one as,
sh(" chmod +x ./build.sh && ....

Reply:
done.